pyatlan/model/assets/g_c_s_object.py

```diff
@@ -416,18 +443,28 @@ class Attributes(GCS.Attributes):
         @classmethod
         @init_guid
         def create(
```

Comment:
again I don't think we should modify deprecated methods.

Reply:
Its `GCSObject.Attributes.create()` method.